CHANGELOG.md

```diff
@@ -41,6 +41,7 @@ IMPROVEMENTS:
  * jobspec: Added support for `headers` option in `artifact` stanza [[GH-9306](https://github.com/hashicorp/nomad/issues/9306)]
 
 __BACKWARDS INCOMPATIBILITIES:__
+ * api: `/v1/event/stream` parameters and API package now use Evaluation and Allocation instead of shortened name. [[GH-9509](https://github.com/hashicorp/nomad/issues/9509)]
```

Comment:
We don't actually have a place for things that change between prereleases and GA other than the email. I'll make sure to not this in the email, but it doesn't need to be in CHANGELOG.md since event stream is a new feature in 1.0.

```suggestion
```

Reply:
sounds good to me!